queries.sql

```diff
@@ -66,4 +66,99 @@ UPDATE animals
 SET species = 'unspecified';
 SELECT species from animals;
 ROLLBACK;
-SELECT species from animals;
\ No newline at end of file
+SELECT species from animals;
+
+ALTER TABLE animals
+DROP COLUMN species;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_owner
+FOREIGN KEY (owner_id)
+REFERENCES owners(id);
+
+ALTER TABLE animals
+ADD COLUMN owner_id integer;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+
+ALTER TABLE animals
+ADD COLUMN species_id integer;
+
+CREATE TABLE species (
+    id serial PRIMARY KEY,
+    name varchar(255)
+);
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name varchar(255),
+    age integer
+);
+
+-- Insert Pokemon
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+-- Insert Digimon
+INSERT INTO species (name)
+VALUES ('Digimon');
+
+-- Insert Sam Smith
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34);
+
+-- Insert Jennifer Orwell
+INSERT INTO owners (full_name, age)
+VALUES ('Jennifer Orwell', 19);
+
+-- Insert Bob
+INSERT INTO owners (full_name, age)
+VALUES ('Bob', 45);
+
+-- Insert Melody Pond
+INSERT INTO owners (full_name, age)
+VALUES ('Melody Pond', 77);
+
+-- Insert Dean Winchester
+INSERT INTO owners (full_name, age)
+VALUES ('Dean Winchester', 14);
+
+-- Insert Jodie Whittaker
+INSERT INTO owners (full_name, age)
+VALUES ('Jodie Whittaker', 38);
+
```

Comment:
Kindly that only the query that helps to analyze the data should be put in the `queries.sql` file. Currently, these are used to populate the table and should be put in the `data.sql` file according to the project requirement. Kindly, fix it.

![image](https://github.com/BezzB/vetclinic/assets/101523582/b6c30e18-ee63-475b-987d-e5abc6eb939c)